src/platform/user/profile/vet360/tests/containers/VAPProfileField.unit.spec.jsx

```diff
@@ -160,33 +160,91 @@ describe('<VAPProfileField/>', () => {
 });
 
 describe('mapStateToProps', () => {
-  const showValidationModalState = () => ({
-    featureToggles: { vaProfileAddressValidation: true },
+  const getBasicState = () => ({
```

Comment:
Sorry this diff is confusing. I actually moved the `showValidationModal` helper down _into_ the `showValidationView` describe block where it's used. This new `getBasicState` helper is used by both the `blockEditMode` and `activeEditView` tests so it lives up at this level.